rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -121,6 +162,30 @@ void SqliteStorage::write(std::shared_ptr<const rosbag2_storage::SerializedBagMe
   write_statement_->execute_and_reset();
 }
 
+void SqliteStorage::write(
+  const std::vector<std::shared_ptr<const rosbag2_storage::SerializedBagMessage>> & messages)
+{
+  if (!write_statement_) {
+    prepare_for_writing();
+  }
+
+  activate_transaction();
+
+  for (auto & message : messages) {
+    auto topic_entry = topics_.find(message->topic_name);
+    if (topic_entry == end(topics_)) {
+      throw SqliteException(
+              "Topic '" + message->topic_name +
+              "' has not been created yet! Call 'create_topic' first.");
```

Comment:
```suggestion
      std::stringstream error_msg;
      error_msg << "Topic '" << message->topic_name << "' has not been created yet! Call 'create_topic' first.";
      throw SqliteException{error_msg.str()};
```

Reply:
This pattern is currently used throughout the file. I think we can open a follow-up style PR for this.